### PR TITLE
Feature 46 add org logos

### LINKED ISSUE #46: Add organization logos
### Description

Add org logos (will be provided for)

### Motivation

Would like to add org logos on the login page with the help of the environment variable wrapper.

### PR DESCRIPTION
**Pull request by nimishbongale**
Closes #46 

- As of now, 5 images are supported, would need a css-grid to accomodate more
- Images get picked dynamically from the images folder
- Controlled by environment variables
- If more images are added, the images would auto-adjust

image

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -10,4 +10,9 @@ REACT_APP_config_path = /opt/rucio/lib/rucio/web/ui/common/saml/
 
 [oidc]
 REACT_APP_idpsecrets = /opt/rucio/etc/idpsecrets.json
-REACT_APP_admin_issuer = wlcg
\ No newline at end of file
+REACT_APP_admin_issuer = wlcg
+
+[login_page_org_images]
+REACT_APP_login_page_org_image_1 = ./images/favicon.png
```
Comment (maany) [inline .env.template:16]: Let's say that the Rucio logo is always going to be present. In addition, Let's just use 2 images, `REACT_APP_login_page_image_primary` and `REACT_APP_login_page_image_secondary`.
We can render all images in the order ( L-R) in a single row : primary -> rucio_logo -> secondary
@bari12 @cserf @mlassnig any comments?

**Review by maany — COMMENTED** [review]
Thanks a lot @nimishbongale , It's cool to support up to 5 images, but I believe we could keep it a bit simpler. ( Also presenting too many configuration options to the end-users might not play well for a user PoV)

File: .env.template (hunk 1 above)
Comment (nimishbongale) [reply .env.template]: Sounds good to me, made the changes!

**Comment by nimishbongale** [comment]
image

(ATLAS image as primary)

image
(ATLAS as secondary)

image
(ATLAS as primary+secondary)

**Review by maany — APPROVED** [review]

**Pull request merged**